uwblueprint/childrens-aid-society
# Odi/visit rest put

## Notion ticket link

[Implement visit REST endpoint (PUT)](https://www.notion.so/uwblueprintexecs/Implement-Visit-REST-Endpoints-PUT-28219f45340042869814846efea40069?pvs=4)



## Implementation description
* Visit endpoint for PUT request created 
* Implemented update_attendance_sheets and update_attendance_records functions 



## Steps to test
1. Check if on put request, date (for records) and any of ["csw", "cpw", "fcc"] (for sheets) have changed
2. Verify that no other parameters have changed 


## What should reviewers focus on?
* What was mentioned above 


## Checklist
